integration/build_dependencies_test.go

```diff
@@ -17,14 +17,97 @@ limitations under the License.
 package integration
 
 import (
+	"fmt"
 	"strings"
 	"testing"
 
+	"github.com/google/uuid"
 	"github.com/sirupsen/logrus"
 
 	"github.com/GoogleContainerTools/skaffold/integration/skaffold"
 )
 
+func TestBuild_WithCache(t *testing.T) {
+	MarkIntegrationTest(t, CanRunWithoutGcp)
+
+	tests := []struct {
+		description string
+		rebuilt     []int
+		cached      []int
+		change      []int
```

Comment:
Reorder these.  Isn't `cache = images - rebuilt`?
```suggestion
		change      []int
		rebuilt     []int
		cached      []int
```

Reply:
fixed